cmd/systray/systray.go

```diff
@@ -103,6 +104,9 @@ func onExit() {
 }
 
 func main() {
+	runtime.LockOSThread()
```

Comment:
Can you add a comment here with a reference to this: https://github.com/lxn/win/commit/d9566253ae00d0a7dc7e4c9bda651dcfee029001 ? 🙇